tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "resolveJsonModule": true,
     "pretty": true,
     /* Basic Options */
-    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es2019",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
I like to use Typescript's optional chaining so targeting `es2019` instead. With `esnext` transpiler doesn't handle optional chaining correctly.